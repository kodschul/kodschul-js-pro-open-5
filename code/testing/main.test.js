const { sum, multiply, createUser } = require("./main");

describe("Math", () => {
  it("should add 1 + 1", () => {
    expect(sum(1, 1)).toBe(2);
    expect(sum(null, null)).toBe(0);
    expect(sum(-100, 100)).toBe(0);
    expect(sum(10000000000000, 10000000000000)).toBe(20000000000000);
  });

  //   it("should multiply 2 by 2", () => {
  //     expect(multiply(2, 2)).toBe(4);
  //   });

  //   it("should create user", () => {
  //     let user = createUser("A B");

  //     expect(user).toMatchSnapshot();
  //   });
});
