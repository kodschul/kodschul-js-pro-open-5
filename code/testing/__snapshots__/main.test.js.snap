// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main should create new user 1`] = `
{
  "firstName": "John",
  "id": "JohnDoe1",
  "lastName": "Doe",
  "name": "John Doe",
}
`;
